Xamarin.Forms.Xaml.UnitTests/VisualStateManagerTests.xaml.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UnitTests;
+
+namespace Xamarin.Forms.Xaml.UnitTests
+{
+	public partial class VisualStateManagerTests : ContentPage
+	{
+		public VisualStateManagerTests()
+		{
+			InitializeComponent ();
+		}
+
+		public VisualStateManagerTests(bool useCompiledXaml)
+		{
+			//this stub will be replaced at compile time
+		}
+
+		[TestFixture]
+		public class Tests
+		{
+			[SetUp]
+			public void SetUp ()
+			{
+				Device.PlatformServices = new MockPlatformServices ();
+				Application.Current = new MockApplication ();
+			}
```

Comment:
add a TearDown, both of those are static, and could influence the test running next

Reply:
Added the TearDown.